### ISSUE TITLE
Add top level archs key

### ISSUE THREAD

**Issue by eddiezane** [chainguard-dev/melange#178]
We should add a top level `archs: []string` key similar to [apko](https://github.com/chainguard-dev/apko/blob/9358e7424e03e6991535f2efeb2cc6126b876017/pkg/build/types/types.go#L77) to avoid having to pass a flag.

**Comment by kaniini** [comment]
I don't think that this is something we should add to Melange.  In contrast to apko, we want Melange to be opportunistic when building packages, constrained by the `.package.target-architecture` property.

Basically, I think `archs` does the wrong thing here, because it would imply that the melange .yaml files would need to be updated to allow Wolfi to be built on a new architecture, which seems like it would be a lot of work.

I assume the issue is relating to Melange wanting to do multi-arch builds by default?  We can and should change that to `--arch host`.
